Update README and round metadata (#445)
reichlab/variant-nowcast-hub: README.md

### Predicted clades

Each week the hub designates up to nine NextStrain clades, based on the highest reported prevalence of those that
meet the following criteria:

> - accounts for at least 1% of observations across the US

Reviewer: ```suggestion
- accounts for at least 1% of observations across the US in any of the three complete
  [USA/CDC epidemiological weeks](https://ndc.services.cdc.gov/wp-content/uploads/MMWR_Week_overview.pdf)
  (a.k.a. MMWR weeks) preceding the Wednesday submission date
```

> - appears at least twice in any of the three complete

Reviewer: ```suggestion
- has had at least two sequences present in the data across the last three complete
```

Early Monday morning (~3am ET) prior to a Wednesday on which submissions are due, the hub generates a JSON file with two high-level properties:

1. `clades`: an array of [NextClade clade names](https://clades.nextstrain.org/about) that will be accepted in submission files for the upcoming deadline.
2. `meta`: metadata relevant to the upcoming round, including

    - links to the Nextstrain sequence information
    - the Nextclade dataset used to generate the above `clades` array

>     - the total number of sequences used to generate the round's clade list

Reviewer: ```suggestion
    - the total number of sequences from the last three weeks of data used to generate the round's clade list
```
